packages/performance/src/services/oob_resources_service.ts

```diff
@@ -27,6 +28,7 @@ export function setupOobResources(): void {
   if (!getIid()) {
     return;
   }
+  setupTransportService();
```

Comment:
setupOobResources is called when the call to get iid and RC is responsed. Starting the transport cycle here means that the initial time of sending logs is "iid call + RC call + 5.5s". I suggest not including those calls in the initial delay to avoid it from being undeterministic.

Reply:
Moved initialization call to `controllers/perf.ts`.